### PR TITLE
Harder test

### PR DESCRIPTION
**Pull request by elusyo**
* Correct wrong target output to look like wrong source output
* Create a test of verbose to test the output with : 
     - One good output
     - One source that doesn't exist
     - One target that doesn't exist

### CODE REVIEW

File: src/parser.ml [hunk 1]
```diff
@@ -7,12 +7,17 @@ let print_files sources target =
                  |-target: %s\n"
     (String.concat " / " sources) target
 
-
+(* Print wrong target error with cmdliner style *)
+let target_error target =
+  "DEST... arguments: no `"^target^"' file or directory\n\
+  \rUsage: distar [OPTION]... SOURCE... DEST\n\
```
Comment (yurug) [inline src/parser.ml]: Put this part in a `usage` function as it does not only relate to the error message.

File: tests/verbose_output/test.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+set -u
+cd "$(dirname "$0")"
+
+## If verbose works
```
Comment (yurug) [inline tests/verbose_output/test.sh]: I do not understand this comment.

File: tests/verbose_output/test.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+fi
+
+# Source doesn't exist
+../distar -v none.ml target.md 2> source.output
+output=$(diff source.output source.expected)
+
+# Check diff exit code
+if [ $? -eq 0 ]
+then
+    printf "\nBad source - OK "
```
Comment (yurug) [inline tests/verbose_output/test.sh]: This description is a bit cryptic.

File: tests/verbose_output/test.sh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+then
+    printf "\nBad target - OK"
+else
+    printf $output
+    exit 1
+fi
+
+# Source doesn't exist
+../distar -v none.ml target.md 2> source.output
+output=$(diff source.output source.expected)
```
Comment (yurug) [inline tests/verbose_output/test.sh]: Please introduce shell function to avoid code repetition.

File: tests/verbose_output/test.sh [hunk 5]
```diff
@@ -3,52 +3,41 @@
...
-if [ $? -eq 0 ]
-then
-    printf "\nBad source - OK "
-else
-    printf -- "\n"$output
-    exit 1
-fi
+
+# Function to test if the verbose mode prints correct outputs
+verbose_mode_test () {
```
Comment (yurug) [inline tests/verbose_output/test.sh]: Document the arguments of this function.

File: tests/verbose_output/test.sh [hunk 6]
```diff
@@ -3,52 +3,41 @@
...
+    else
+        echo "\n"$output
+        exit 1
+    fi
+}
+
+
+
+# Test with good arguments
+verbose_mode_test 1 "source_1.ml" "target.md" "good_args"
```
Comment (yurug) [inline tests/verbose_output/test.sh]: The captions of these tests are cryptic.

Comment (elusyo) [reply tests/verbose_output/test.sh]: Do I have to precise exactly why it works ?

**Comment by yurug** [comment]
What is "it" in your question?

**Comment by elusyo** [comment]
The sentence but I replace this comment by a better explanation

**Pull request merged**